fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -323,12 +323,24 @@ def __init__(
         self.process_group = process_group or get_process_group_cached()
         # If ProcessGroupName.default is passed in, the reduce_scatter will use the same process group with
         # the rest of operations. The overlap feature in the backward propagation is disabled.
+        # If ProcessGroupName.reduce_scatter is passed in, the reduce_scatter use a seperate process group
```

Comment:
I would move the comment to the if/else statement that it applies to.

Reply:
will do.